crates/compiler/build/src/program.rs

```diff
@@ -384,6 +397,8 @@ pub fn gen_from_mono_module_llvm(
             }
             _ => unreachable!(),
         }
+
+        MemoryBuffer::create_from_file(&app_o_file).expect("memory buffer creation works")
```

Comment:
Why is the memory buffer still from a file? Is this basically mmaping the file?

Reply:
could have used `std::fs::read` here I guess